mlir/lib/Target/LLVMIR/DebugTranslation.cpp

```diff
@@ -102,7 +102,9 @@ llvm::DICompileUnit *DebugTranslation::translateImpl(DICompileUnitAttr attr) {
       attr.getSourceLanguage(), translate(attr.getFile()),
       attr.getProducer() ? attr.getProducer().getValue() : "",
       attr.getIsOptimized(),
-      /*Flags=*/"", /*RV=*/0);
+      /*Flags=*/"", /*RV=*/0, /*SplitName=*/{},
+      static_cast<llvm::DICompileUnit::DebugEmissionKind>(
```

Comment:
I wonder why we did not choose an LLVM enum here... That way we would have a guarantee that the two enum values match. However, this is orthogonal to your revision.

Reply:
Yeah there are other enums like `DISubprogramFlags` that are also just redefined and rely on static_cast. Might be a good idea to reuse llvm enums if possible, or set up explicit mappings?

Reply:
Yeah I guess I would prefer if we use LLVM enums since there are no large switch statements needed (I assume that is what you refer to by explicit mappings)? But this is unrelated to your PR. I just realized that things are a bit inconsistent in that regard.

Reply:
Yeah that's what I meant. That makes sense. We can leave it at this then for this PR. Can you help me click merge once it's ready? Thanks! 🙏

Reply:
sure!